apprise_api/api/templates/base.html

```diff
@@ -42,7 +42,7 @@ <h1>{% trans "Apprise API" %}</h1>
       <div class="col s3">
 {% if STATEFUL_MODE != 'disabled' %}
         <ul class="collection z-depth-1">
-          <a class="collection-item" href="{% url 'config' 'apprise' %}"><i class="tiny material-icons">settings</i>
```

Comment:
I don't think this is necessary... if anything i should have used this line throughout all of the code.